src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const obj in action.extraData) {
```

Comment:
```suggestion
        for (const key in action.extraData) {
```
Там скорее ключ, чем целый объект